eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,47 @@
+trigger: none
+
+pr:
+  paths:
+    include:
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: ubuntu-22.04
```

Comment:
Please reference https://github.com/Azure/azure-sdk-for-js/blob/main/eng/pipelines/templates/variables/image.yml and use the pool and image variables defined there.

Reply:
Done.